web/moduleMapper/dummy.tsx

```diff
@@ -88,17 +88,79 @@ export function k8sUpdate(k8s: any): Promise<any> {
   return Promise.resolve(k8s);
 }
 
+export function k8sDelete(k8s: any): Promise<any> {
+  console.log("k8sDelete", k8s);
```

Comment:
is that temporary debug log or something we want to keep?

Reply:
The dummy file is not imported on production builds since it's part of console api. It's useful to see what's been sent in standalone builds.

https://github.com/netobserv/network-observability-console-plugin/blob/main/web/webpack.standalone.js#L81